sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -320,6 +320,8 @@ export class SessionReceiver {
 
   /**
    * @property {Date} [sessionLockedUntilUtc] The time in UTC until which the session is locked.
+   * This is defined only when the underlying messageSession has been created.
+   * The value until which it would be locked until is 30 seconds.
```

Comment:
This documentation will be visible to our users who are not aware of what a "messageSession" is.
"messageSession" is a internal construct that we use in the sdk.
You can refer to "AMQP receiver link" instead.